MediaBrowser.Server.Mono/EmbyServer.csproj

```diff
@@ -25,12 +25,6 @@
     <PackageReference Include="SQLitePCLRaw.core" Version="1.1.11" />
     <PackageReference Include="SQLitePCLRaw.provider.sqlite3.netstandard11" Version="1.1.11" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.5.0" />
-    <Reference Include="Emby.Server.Connect">
-      <HintPath>..\ThirdParty\emby\Emby.Server.Connect.dll</HintPath>
-    </Reference>
-    <Reference Include="Emby.Server.Sync">
-      <HintPath>..\ThirdParty\emby\Emby.Server.Sync.dll</HintPath>
```

Comment:
What was the Sync feature anyway?

Reply:
AFAIK. It was a way to sync media to apps.

Reply:
So the sync function will be removed for now? That's fine, just might have to remove some UI stuff as well. The sync function should honestly be handled client-side anyways, it was basically an option to select certain media and always have it stored locally on the client. Basically a download but also handles changes to the media, should be a client-side check.

Reply:
We can think of re-implementing some subset of Sync, maybe in a form of "download" link (potentially with transcoding to something that client device can handle)...